types/basic_value_descriptions.h

```diff
@@ -36,7 +36,7 @@ struct DefaultDescription<Datacratic::Id>
             val->val2 == 0 && val->val1 <= std::numeric_limits<int32_t>::max()) {
             context.writeInt(val->val1);
         } else {
-            context.writeString(val->toString());
+            context.writeStringUtf8(Utf8String(val->toString()));
```

Comment:
This one is a bit icky, but OK I guess.  What we're doing here is saying that Ids have UTF8 characters implicitly.  I'd much rather that was explicit.